bzt/modules/apiritif/generator.py

```diff
@@ -549,6 +562,7 @@ def _gen_wait_sleep_mngr(self, atype, tag, param, selectors):
         mode = "visibility" if param == 'visible' else 'presence'
 
         if atype == 'wait':
+            self.log.warning("Wait command is deprecated and will be removed soon. Use waitFor instead.")
```

Comment:
what do you think about joining implementation of 'wait' and 'waitfor'? something like
```
if atype.startswith('wait'):
  if not atype.endswith('for'):
    warning(...)
    <convert wait params to fit for waitfor>
  _gen_wait_for_mngr(...)
```
imho it allows to avoid code duplication

Reply:
Yes, it makes sense to me...